src/Components/SceneBreadcrumb/SceneBreadcrumb.types.ts

```diff
@@ -15,6 +15,7 @@ export interface IBaseBreadcrumbProps {
     classNames: BreadcrumbClassNames;
     onSceneClick?: () => void;
     onCancelForm?: () => void;
+    onCloseBehaviorsModal?: () => void;
```

Comment:
onSceneChange?

Reply:
I'll think of another name then, maybe `triggerCloseBehaviorsModal` because `onSceneChange` is too generic and this is specific to the modal in View mode. The modal is not present in Build mode.

Reply:
I see what you mean, will make the change